mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Koans.Mixfile do
   def project do
     [app: :elixir_koans,
      version: "0.0.1",
-     elixir: ">= 1.2.1 and <= 1.4.0",
+     elixir: ">= 1.2.1 and <= 1.4.1",
```

Comment:
Good point.  I don't know semver requirement syntax very well so here's the best I could come up with: `~> 1.2.1 or ~> 1.3.0 or ~> 1.4.0`

This would allow everything from `1.2.1`, through the `1.3` and `1.4`  minor releases regardless of the patch version as you can see:

<img width="502" alt="screen shot 2017-01-30 at 15 57 08" src="https://cloud.githubusercontent.com/assets/3731516/22430087/d853a33e-e704-11e6-820e-8d37527af03f.png">

_Off topic: how cool is that `Version` module?_

I remember something breaking on `1.2.0` which is why the low end cares about the patch version.